portmap/core/admin.py

```diff
@@ -61,3 +63,20 @@ class CustomUserAdmin(UserAdmin):
 @register(DataType, site=admin_site)
 class DataTypeAdmin(admin.ModelAdmin):
     list_display = ('id', 'name', 'helpText')
+
+@register(TrackArticleView, site=admin_site)
+class TrackArticleViewAdmin(admin.ModelAdmin):
+    list_display = ('article_path', "count", "visited_directly", "external_referrer",)
+    aggregated_counts = {}
+
+    def count(self, object):
+        matched_item = next(item for item in self.aggregated_counts if item["article_path"] == object.article_path)
+        return matched_item["count"]
+
+    def get_queryset(self, request):
```

Comment:
You shouldn't need to do get_queryset or get_object here, especially if the count is moved to Article list display